templates/core/crowdfunding_donors.html

```diff
@@ -0,0 +1,49 @@
+{% extends 'global/base.html' %}
+{% load staticfiles %}
+
+{% block content %}
+<div class="row">
+    <div class="col-md-1"></div>
+    <div class="col-md-10">
+    <h2>Special thanks to all our amazing supporters!</h2>
+        <div class="row">
+        <div class="col-md-6">
+            <p>This year we ran a crowdfunding campaign on Indiegogo's generosity website from the 29th January
+                to the 29th of March with the aim of raising US $10,000 to help us cover our staff and
+                operational costs for the year.
+                We are truly grateful to all our amazing supporters who helped reach and surpass our target goal.
+                We managed to raise a total of $13,314!</p>
+
+            <p>If you missed our campaign but will still like to support our work, you can donate to us via
+                PayPal or credit/debit card for once off donations. Alternatively, you can sign up on <a href="http://patreon.com/djangogirls">Patreon
+                </a> for monthly donations. Thank you for supporting our work.</p>
+
+            <p>Thank you. Your support means the world to us.</p>
+        </div>
+        <div class="col-md-6">
+                    <img class="img-responsive center-block" alt="Django Girls Gliwice #1." src="{% static 'img/global/donate/tnr.jpg' %}">
+                </div>
+        </div>
+
+        <div class="row">
+            <h2>They Support Us</h2>
+            {% for donor in donors %}
```

Comment:
I think we need to add some styles to make donors look a bit nicer: now it's just a simple columns with donor's names. Maybe we could add our Django Girls heart near each name? Or choose some nice font?